### PR TITLE
login_frontend

### PR DESCRIPTION
**Pull request by teemul491**
recieving error as it is not posting the request

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
 import React from "react";
-import TodoList from "./components/TodoList";
+// import TodoList from "./components/TodoList";
```
Comment (kotAPI) [inline src/App.js]: you can delete these commented lines if you don't need them

File: src/components/TodoForm.js [hunk 2]
```diff
@@ -36,7 +36,11 @@ function TodoForm(props) {
             ref={inputRef}
             className="todo-input edit"
           />
+<<<<<<< Updated upstream
```
Comment (kotAPI) [inline src/components/TodoForm.js]: these are merge conflicts that need to be fixed before merging

File: src/components/login/login.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        const { name, value } = e.target
+        setUser({
+            ...user,
+            [name]: value
+        })
+    }
+
+    return (
+        <div className="login">
+            {console.log(user)}
```
Comment (kotAPI) [inline src/components/login/login.js:20]: remove logs

File: src/App.js (hunk 1 above)
Comment (Abhishekcs194) [reply src/App.js]: They have been commented for testing purposes
will have to uncomment

File: src/components/TodoForm.js (hunk 2 above)
Comment (Abhishekcs194) [reply src/components/TodoForm.js]: this is resolved

**Pull request closed without merging**